hrjobcontract/js/src/job-contract/services/contract-leave.js

```diff
@@ -14,6 +14,18 @@ define([
             json: {}
         });
 
+        /**
+         * The API returns values as strings, so we convert them to booleans to
+         * make it easy to use them inside conditions
+         *
+         * @param {Object} values - The values object as returned by the API
+         */
+        function adjustAddPublicHolidaysValue(values) {
+          angular.forEach(values, function (value) {
+            value.add_public_holidays = !!parseInt(value.add_public_holidays);
```

Comment:
It is. The API returns the number as a string (`'1'` or `'0'`) and the coercion alone would return true for both:
![captura de tela 2016-07-07 as 07 18 10](https://cloud.githubusercontent.com/assets/388373/16650158/0aca6680-4413-11e6-8f45-ac71c535f78d.png)